src/common/transactions/transaction-utils.ts

```diff
@@ -119,3 +128,67 @@ export const generateSTXTransferTx = ({
 
 export const stacksTransactionToHex = (transaction: StacksTransaction) =>
   `0x${transaction.serialize().toString('hex')}`;
+
+export const isContractFtTokenTransfer = (tx: Tx) => {
+  return (
+    tx.tx_type === 'contract_call' &&
+    tx.contract_call.function_name === 'transfer' &&
+    tx.contract_call?.function_args?.[0]?.name === 'amount'
+  );
+};
+
+// calculate the real amount of the token based on the decimal number
+// specified in the corresponding token smart contract
+export const tokenRealAmount = (asset: AssetWithMeta | undefined, tx: ContractCallTransaction) => {
```

Comment:
This function name isn't very clear. There could be a misunderstand as to what "real" means.

How about `calculateTokenTransferAmount`? Open to other suggestions.

Reply:
I agree on this, not understanding what `real` means right away.